test/vm_tests/transformer/impl_generation_test.dart

```diff
@@ -1009,7 +1009,10 @@ main() {
         MethodDeclaration propsImplCall = propsImplClass.members
             .singleWhere((entity) => entity is MethodDeclaration && entity.name?.name == 'call');
 
-        expect(propsImplCall.parameters.toSource(), uiPropsCall.parameters.toSource(),
+        var generatedParameterList = uiPropsCall.parameters.parameters.expand((param) => [param.identifier.name]);
+        var expectedParameterList = uiPropsCall.parameters.parameters.expand((param) => [param.identifier.name]);
+
+        expect(generatedParameterList.toString(), expectedParameterList.toString(),
```

Comment:
We actually should also be able to remove these generated `call` declarations for the same reason as for `DomProps`/`SvgProps` above!